src/Configuration.ts

```diff
@@ -46,7 +46,7 @@ export const CONFIG = {
       IDENTIFICATIVO_PSP: "AGID_01",
       IDENTIFICATIVO_INTERMEDIARIO_PSP: "97735020584",
       IDENTIFICATIVO_CANALE: "97735020584_02",
-      TOKEN: process.env.PAGOPA_TOKEN || "ND"
+      PASSWORD: process.env.PAGOPA_PASSWORD || "nopassword"
```

Comment:
without the pagopa password the system can't work and should fail to start, it doesn't make sense to have a default password, does it?

Reply:
It's true. I did it because password is defined as "NonEmptyString".
It's a default value to start server without the need to set env variable (password is not used by mock server).

Do you prefer to remove it anyway?

Reply:
yes I would prefer the server to fail to start if no password gets set - this is a safe behavior in case we deploy the server to production and we forget to set the password